rust-toolchain.toml

```diff
@@ -1,4 +1,4 @@
 [toolchain]
-channel = "1.78.0"
+channel = "1.76.0"
```

Comment:
Why is `dlmalloc` failing? We can debug that, rather than downgrading the rust version

Reply:
to fix this you need to upgrade wasm-bindgen to `2.92.0` or higher, and once you do that web-sys starts failing to compile and then we need to fix that then the chain continues